src/PhpSpreadsheet/Worksheet/Worksheet.php

```diff
@@ -3442,7 +3442,7 @@ public function getHashCode()
      * @param string $range Range to extract title from
      * @param bool $returnRange Return range? (see example)
      *
-     * @return ($range is non-empty-string ? ($returnRange is true ? array{0: string, 1: string} : string) : ($returnRange is true ? array{0: null, 1: null} : null))
+     * @return ($range is non-empty-string ? ($returnRange is true ? array{0: string, 1: string} : string) : ($returnRange is true ? array{0: null, 1: null} : null)) Scrutinizer does not understand this but it helps Phpstan
```

Comment:
```suggestion
     * @return ($range is non-empty-string ? ($returnRange is true ? array{0: string, 1: string} : string) : ($returnRange is true ? array{0: null, 1: null} : null))
```

Don't think we should have those kind of comment that end up in end-user documentation.